barretenberg/cpp/src/barretenberg/stdlib/primitives/address/address.hpp

```diff
@@ -3,8 +3,7 @@
 #include "barretenberg/stdlib/commitment/pedersen/pedersen.hpp"
```

Comment:
nit: we should nuke this file from barretenberg and put it in circuits -- I can do it in a followup PR